project-communications/newsletter/newsletter-012921.md

```diff
@@ -0,0 +1,181 @@
+
+Best wishes for everyone!
+
+## Project News
+
+### Meet-up sessions
+
+We had our first open online meet-up session last Thursday (25th January). 
+This included a wonderful set of developers, researchers, and autistic collaborators with a whole variety of skills and perspectives. 
+Thank you everyone who took part! 
+
+The next meet-up session will take place on **Thursday 11th February** at an earlier time to accommodate more people. 
+Agenda and details to follow. 
+
+### We are on Twitter! 
+
+AutSPACEs now has a Twitter account! Follow us: https://twitter.com/AutSpaces!
+
+If you are social media savvy we would love to hear from you and get input into running the account, sharing it, and using it to promote the platform. 
+
+## Get involved
+
+Following the first meet-up group session on Thursday 25th January, a number of areas crucial to the project were identified in which people were interested in helping out:
+
+* Groups
+	* Research 
+	* Community engagement
+	* Promoting the platform 
+	* Web development  
+
+If you would like to join onee of these groups and meet and work with other autistic people, or to suggest another working group, please email Georgia at gaitkenhead@turing.ac.uk.
+
+### Take part in the ACCEPT study
+
+A team of clinicians, researchers and experts by experience are running a project to understand and share ways to help health, social care and public facing workforces across England to work with autistic people more effectively. 
+We want to help them make use of the Core Capabilities Framework for Supporting Autistic People (2019). 
+You can help by filling in the [ACCEPT survey](https://kclbs.eu.qualtrics.com/jfe/form/SV_cUxOb07WGQl6IO9?dm_i=4U0R,GIAA,2B1RTB,1Y9R6,1). 
+
+## Presenting the project
+
+The project is being presented on the **26th February from 2pm-3pm** for the *Social Data Science Spcecial Interest Group* at The Turing. 
+It is not a public event, but if you are interested in joining just email Georgia at gaitkenhead@turing.ac.uk and she will link you to the event. 
+
+## Introducing Alice
+
+Alice recently joined the Autistica/Turing citizen science project and has already launched herself into collaborating, co-facilitating our first online meet-up session and brainstorming strategies of engagement and areas of research for the project. 
+She is also founder, director, and CEO of a social purposes organisation based in Brighton and working across the South-East. 
+It works with young refugees and asymlum seekers. 
+Alice founded it almost 20 years ago. 
+The organisation is  called 'Pathways to Independence'. 
+It provides support and accommodation to young refugees and asylum seekers, unaccompanies 16+. 
+You can find our more on the organisation's [website](https://www.pathwaystoindependence.org.uk/).
+
+Here she is; 
+
+![](https://i.imgur.com/jJUd1uJ.jpg)
+
+And here she is in her own words: 
+
```

Comment:
Just a bit of clean up formatting (there were floating >’s and a few extra quotation marks, and some spelling checks” 

Hear about Alice and her work with us here: 


"I'm a 47 year-old late-diagnosed autistic woman and mother. I am actually multiply neurodivergent: I'm autistic, and I have attention differences. My proudest achievement is that I'm a Mum to my daughter Isabell, who is also multiply neurodivergent. We live together in our home in Brighton with our cat Belle, who I say to my daughter is also autistic - and that's why I picked her, because she's very particular."


"I think something interesting about my profile, which I think is becoming more and more common now, is I've lived life both sides of the river. I've lived for 46 years as a neurotypical person, and I've only lived the past year and a half knowing that I was autistic. Getting a diagnosis was life-changing for me. My motivation wasn't because I needed extra support, it was more because of curiosity, and because of my daughter. My diagnosis came as a positive thing, as a quest of curiosity and self-knowledge, rather than being because I was experiencing problems in my life. So that's actually quite an unusual thing. I wasn't "diagnosed" in that pathology paradigm of something being wrong. So for me, I think that's quite an interesting angle, that my diagnosis was about something more, it was about my strengths - but also so that I could locate some of the challenges that I'd previously experienced in my neurotypical life."

"For me it's just a different way of being in the world, which describes my relationship to the world and people. I always grew up feeling different, and not really understanding why. I had different feelings about things, different reactions to things, and I felt like a refugee actually - which is quite interesting given my work. I felt dislocated and alienated. So when I got my diagnosis I could make sense of why I felt and thought differently. It was validating. A lot of the things I would criticise myself for, like that I couldn't understand things in the same way other people did, or that I couldn't think about things in the same way - can lead to really negative feelings about yourself, which can become a self-perpetuating cycle."

"Some autistic people have quite switched off experiences, but for me, I experience life in glorious technicolour. Don't get me wrong, I'm not seeing things dancing - it's rooted in reality, but I see and feel things very intensely. My particular profile is I'm hyper-sensitive and hyper-emotional. So an example of that would be that if I saw a suffering animal or a dead animal, I would experience pain. My upset will manifest in internal pain, and it will literally be like experiencing high emotions and pain for a couple of days. You're carrying this trauma of this animal with you. So that's an example of how I might experience an event - or something that's difficult or sad. Being autistic can make me different to people, so it can disconnect me - but also, for me, I feel really proud to be autistic. It's given me strength and it's given me power, and being autistic explains my success in life. I'm autistic with attention deficiencies, so for example the impulsivity associated with ADHD can lead some people into negative, painful experiences. That impulsivity led me into something really positive - into risk-taking, because I thought, 'yeah, I'm going to set my own business up'."

"For me, there's both 'despite of', and 'because of' with autism. Being autistic has led to challenges, but it's also opened doors to things and people and worlds I would never have imagined. I feel that it's something that I need to protect and honour and really take care of. It's like harbouring a very precious vase inside of you that you have to really take care of. When I got my diagnosis, it lead to compassion, so my diagnosis gave me self-understanding, and kindness to myself, whereas before I used to be very self-critical, feeling like I was doing things wrong. It was like looking in a kaleidoscope and then being able to switch my whole life, and see my life in a whole new way, going forwards."

"My mates who are in the autistic community are trashing the old pathology paradigm around theory of mind and lack of empathy. Yes, there are some individuals on the spectrum, as there are some individuals not on the spectrum, who may have mental illness and may have a lack of empathy, but then there are people on the spectrum, just like there are people not on the spectrum, who have hyper-empathy, and in fact there's a huge emerging area of research - you have hypo and hyper - you can be more "switched off" and more "switched on", like me, and both can be a gift and a burden. It can be challenging, which is why you need to protect and look out for yourself. So I am of that school of thought, that every day lay notions of autism or autistic people lacking empathy or social skills, it's really old - it's dead."

"With courage, I'm talking to people like you. I'm trying to raise awareness and change the profile, so that across communities and society there are changes in how we perceive neurodiversity. People like me, I have great social skills - I maybe talk too much, but I communicate highly effectively. I'm highly articulate, I have emotionality - a huge emotional landscape, so that I was able to found and sustain a social organisation working with loads of professionals who might have diagnosed me - doctors, psychiatrists, social workers."

"The circles that I'm moving in: I'm moving in academic circles, because of my masters. I'm moving increasingly in professional circles. I am seeing the embracing of the new paradigm. But that has to filter down. So they are the thinkers and the doers in terms of structure. Where I don't see it is in the non-autistic community around me. That's not the case with my friends, luckily - probably because all my friends are neurodiverse, either diagnosed or undiagnosed. Also the kinds of people I would be friends with anyway are really open-minded. But I suppose as I'm navigating day to day life, sadly, there's still these really old misconceptions about autism amongst even the middle class intelligentsia, teachers, and all kinds. There's a lot of work to be done. But I do see it in academic and in organisational and professional circles."

"I wonder, and I think maybe in 50 years time neurodivergent people are going to be the new neurotypicals, because it's such an emerging and growing area, and realising that human nature, what it is to be human, is not singular, and it's not binary. It's not, you know, male and female - gender is a huge construct, ethnicity, race. It's the same with neurodiversity. It's complex, and what it is to be human is not singular."

"That is why intersectionality is so important, and neurology with other categories and dimensions, such as gender and class. Some of what I've been doing in my master's is around reframing and reclaiming those spaces, so you're not just autistic, you're, for instance, neuro-queer. So taking derogatory or pathologizing language and reclaiming it. That's what I'm really interested in, in terms of my research - all this myriad, and kaleidoscope - plus, plus, plus, plus, plus.”

"How I found out about this project was that it came up on Facebook. I'm invested in a lot of autistic organisations, such as The National Autistic Society and Autistica. I'd already looked at Autistica's website and seen a lot of what they did, and I've signed up to be involved in research. I feel this is a natural synergy. This popped up at around 6:30 which is a really random time for me even to be looking at stuff, and I was making something for my daughter, and I thought, "wow, this is interesting". Particularly because the whole sensory aspect of being neurodivergent is massively neglected. I listened to the talk. There were two bits of this: the fact that it's sensory and the participatory aspect - and then the extension of that, which is citizen science - I thought, 'wow'. I like anything which is pioneering, anything that's cutting edge. Because I'm a CEO, I like seeing patterns of what's new and emerging. So I'm involved because I feel that this project is in an exciting, emerging area, and because we know that this is what autistic people have told us feels important to them. It's massively under-researched. So I thought, "yes, I really, really want to do this, I really want to participate in this". I actually feel that it's a commitment and a responsibility to change things for other people, so I really want to be involved."

Reply:
I missed a grammar point in paragraph 5 

"For me, there's both 'despite of', and 'because of' with autism. Being autistic has led to challenges, but it's also opened doors to things and people and worlds I would never have imagined. I feel that it's something that I need to protect and honour and really take care of. It's like harbouring a very precious vase inside of you that you have to really take care of. When I got my diagnosis, it led to compassion, so my diagnosis gave me self-understanding, and kindness to myself, whereas before I used to be very self-critical, feeling like I was doing things wrong. It was like looking in a kaleidoscope and then being able to switch my whole life, and see my life in a whole new way, going forwards."

Reply:
and "theory of mind" should be capitalized (paragraph 6) 

My mates who are in the autistic community are trashing the old pathology paradigm around Theory Of Mind and lack of empathy.